app/components/post_draft/read_only/read_only.test.js

```diff
@@ -4,18 +4,18 @@
 import React from 'react';
 
 import Preferences from '@mm-redux/constants/preferences';
-import {renderWithIntl} from 'test/testing_library';
 
+import {shallowWithIntl} from 'test/intl-test-helper';
 import ReadOnly from './index';
 
 describe('PostDraft ReadOnly', () => {
     test('Should match snapshot', () => {
-        const wrapper = renderWithIntl(
+        const wrapper = shallowWithIntl(
             <ReadOnly
                 theme={Preferences.THEMES.default}
             />,
         );
 
-        expect(wrapper.toJSON()).toMatchSnapshot();
+        expect(wrapper).toMatchSnapshot();
```

Comment:
why this change?

Reply:
@saturninoabril thoughts?

Reply:
One option is to mock it, like
```javascript
jest.mock('react-native-vector-icons', () => {
    const {Text} = jest.requireActual('react-native');

    return {
        createIconSetFromFontello: () => {
            function IconSet(props) {
                return <Text {...props}>{'Compass Icon'}</Text>; // added text to indicate name in the snapshot
            }
            return IconSet;
        },
    };
});
```
I'm not fully convince if the mock up is worth doing but it would be good not to go back with `enzyme` in favor of `testing_library` as much a possible. I'll spend time to investigate more to get better options. For the meantime, I'm good as is whichever is implemented.